dtfs-central-api/src/types/utilisation-reports.d.ts

```diff
@@ -48,3 +49,9 @@ export type UpdateUtilisationReportStatusInstructions = {
   status: UtilisationReportReconciliationStatus;
   filter: ReportFilter;
 };
+
+type UtilisationReportHeader = ValuesOf<typeof UTILISATION_REPORT_HEADERS>;
+
+export type ReportData = {
```

Comment:
As this type is now in a general types file could we give it a less general name? Maybe `UtilisationReportRawCsvData` to distinguish from other data?